addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -47,6 +47,18 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         };
         this.$body[0].ownerDocument.addEventListener('selectionchange', this.__onSelectionChange);
 
+        // Even if we prevent the drag via the css, we have to override the
+        // dragstart event because if one of the image ancestor has a dragstart
+        // listener, the dragstart handler can be called with the image as
+        // target.
+        this.__onDragStart = ev => {
+            if (ev.target.matches("img")) {
+                ev.preventDefault();
+                ev.stopPropagation();
+            }
+        };
+        this.$body[0].addEventListener("dragstart", this.__onDragStart);
```

Comment:
It's strange that we need that.. I can reproduce the issue in a code sandbox as you can see here.
![image](https://github.com/odoo/odoo/assets/78849981/b72cd985-0128-47eb-854d-29134745e678)

Seems to be a strange behavior of browser.

Reply:
BVR and I thought you might have an opinion on this :arrow_up: @qsm-odoo ?

Reply:
I don't know, what is the question? :wink: 
- user-drag does not seem support in CSS
- can't `draggable="false"` be used? (or does it not work either?)

Reply:
Agreed, do what you think is best :+1: